docs/add_logging.md

```diff
@@ -38,3 +38,44 @@ public class MyClass
     }
 }
 ```
+
+### Additional Configuration
+
+The logging system has several configuration options that are provided to make it easier
+to use in cloud native environments.
+
+* `log.output-file` can be set to the desired output for logging and also provides a mechanism
+  to stream logs over tcp. Setting this to the format `tcp://<host>:<port>` will enable the socket
+  logger for tcp streaming.
+
+* `log.format` can be set to either `text` or `json`. When set to `json` all fields are encapsulated
+  in potentially multi-line `json` fields. This allows for capturing and indexing exceptions as singular fields in
```

Comment:
Is the multi-line part correct? We don't enable indentation, so the JSON itself shouldn't have newlines, and the string values should have newlines escaped. Maybe rephrase as

> When set to `json`, the log record is formatted as a JSON object, one per line. Any newlines in the field values, such as exception stack traces, will be escaped as normal in JSON object. This allows ...

Reply:
Nit: write the second `json` instance as JSON without code quotes.